cmd/workaround_flannel_issue_1044.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/platform9/nodeadm/constants"
+)
+
+// ensureFlannelDaemonSetTolerations patches the flannel daemonset so that it
+// tolerates all NoSchedule taints. See https://github.com/coreos/flannel/issues/1044
+func ensureFlannelDaemonSetToleratesAllNoScheduleTaints() error {
```

Comment:
Why not call `patchFlannelDaemonSet()` directly, is it to mimic the convention of `ensureKubeProxyRespectsHostoverride()`?

Reply:
Yeah, the former is expected to be idempotent. For the kubeproxy workaround, I needed to make the patch idempotent, but here kubectl makes it idempotent on its own.